docker/Dockerfile.onnx-mlir

```diff
@@ -55,6 +56,10 @@ RUN LLVM_PROJECT_ROOT=${WORK_DIR}/llvm-project \
     && TEST_MCPU=${TEST_MCPU:-$([ "$(uname -m)" = "s390x" ] && echo z14 || \
                                ([ "$(uname -m)" = "x86_64" ] &&  echo || \
                                ([ "$(uname -m)" = "ppc64le" ] && echo || echo)))} \
+# Enable Constant Propagation 
+    && TEST_CONSTANT_PROP=${TEST_CONSTANT_PROP:-$([ "$(uname -m)" = "s390x" ] && echo true || \
+                                                ([ "$(uname -m)" = "x86_64" ] &&  echo true || \
+                                                ([ "$(uname -m)" = "ppc64le" ] && echo true || echo true)))} \
```

Comment:
There is no need for this if this is going to be set the same way on all archs. Simply put
```
ARG TEST_CONSTANT_PROP=true
```
above.

Reply:
Okay let me update that now

Reply:
Updated